rest/tests/test_rest.py

```diff
@@ -0,0 +1,770 @@
+"""
+Unit tests for REST api
+"""
+
+from __future__ import unicode_literals
+
+import json
+from django.test import Client
+from rest_framework.reverse import reverse
+
+from learningresources.tests.base import LoreTestCase
+from learningresources.models import Repository
+from taxonomy.api import create_vocabulary
+from taxonomy.models import Vocabulary, Term
+from rest.serializers import VocabularySerializer
+
+OK = 200
+CREATED = 201
+NO_CONTENT = 204
+BAD_REQUEST = 400
+FORBIDDEN = 403
+NOT_FOUND = 404
+METHOD_NOT_ALLOWED = 405
```

Comment:
I don't think it's necessary going in this direction, but since I didn't know a better solution, I asked on StackOverflow:
http://stackoverflow.com/questions/30846551/how-to-check-http-status-code-in-python-version-agnostic-way